paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -333,6 +339,65 @@ class PaymentSheet internal constructor(
         }
     }
 
+    @Parcelize
+    data class PrimaryButton(
```

Comment:
Should this be PrimaryButtonTheme to avoid confusion with the view?

Reply:
This is a publicly facing API. So it's preferred we keep the name `PrimaryButton` and if there's too much confusion rename the others that are internal.